src/SpannerDriver.php

```diff
@@ -272,6 +266,15 @@ private function getSpannerClientFactory(): SpannerClientFactory
         return $this->spannerClientFactory;
     }
 
+    private function getSessionPoolFactory(): SessionPoolFactory
+    {
+        if ($this->sessionPoolFactory === null) {
```

Comment:
If we migrate to PHP >7.4 and declare property type - we will receive an error here: https://www.php.net/manual/ru/migration74.new-features.php#124654

Maybe it will be better to use `isset`?
Wdyt?

Reply:
Migrated to the constructor instead :)